<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addConfigWindow</class>
 <widget class="QWidget" name="addConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>545</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:8px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_windowTitle" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent; </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_windowTitle">
             <property name="toolTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:bold;
font-size:16px;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>添加配置信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>623</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_menu" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>45</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color:rgb(242, 113, 113); 
image: url(:/png/closeW.png)15 0 0 0;
}
QPushButton { 
image: url(:/png/close.png)15 0 0 0;
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">	background-color: rgb(255, 255, 255);
	border-radius: 8px;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 65, 89);</string>
              </property>
              <property name="text">
               <string>ProductKey</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_pk">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 65, 89);</string>
              </property>
              <property name="text">
               <string>ProductSecret</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ps">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 65, 89);</string>
              </property>
              <property name="text">
               <string>数据格式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_data">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox{
color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
background-color: rgb(245, 245, 245);
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
}
QComboBox:disabled
{
color: rgb(180, 180, 180);
font-size:14px;
border-style:solid;
background-color: rgb(220, 220, 220);
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
}
QComboBox::drop-down{
border-style:none;
width: 30px;
}
QComboBox::down-arrow {
image: url(:/png/QComboBox_down.png);
}

QComboBox QAbstractItemView::item
{ 
	color: rgb(54, 65, 89);
	height: 25px;  
}</string>
              </property>
              <item>
               <property name="text">
                <string>透传数据</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>物模型</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_file">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 65, 89);</string>
              </property>
              <property name="text">
               <string>物模型文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_automatic">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QCheckBox::indicator{
    width: 30px;
    height: 23px;
}

QCheckBox::indicator:unchecked{
    image:url(:/png/checkBox0.png);
}

QCheckBox::indicator:checked{
    image:url(:/png/checkBox1.png);
}
QCheckBox{background-color:transparent;color: rgb(104, 112, 130);}
</string>
              </property>
              <property name="text">
               <string>自动获取</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_environment">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox{
color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
background-color: rgb(245, 245, 245);
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
}
QComboBox:disabled
{
color: rgb(180, 180, 180);
font-size:14px;
border-style:solid;
background-color: rgb(220, 220, 220);
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
}
QComboBox::drop-down{
border-style:none;
width: 30px;
}
QComboBox::down-arrow {
image: url(:/png/QComboBox_down.png);
}

QComboBox QAbstractItemView::item
{ 
	color: rgb(54, 65, 89);
	height: 25px;  
}</string>
              </property>
              <item>
               <property name="text">
                <string>中国数据中心</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>欧洲数据中心</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>美国数据中心</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="lineEdit_file">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_getFile">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_file">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
	padding-left:15px;
	padding-right:15px;
}
QPushButton:hover{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.2);
}</string>
                </property>
                <property name="text">
                 <string>导入本地文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_getFile">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
	padding-left:15px;
	padding-right:15px;
}
QPushButton:hover{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.2);
}</string>
                </property>
                <property name="text">
                 <string>自动获取文件</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(140,140,140);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(140,140,140);
border-radius:4px;
padding-left:15px;
padding-right:15px;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_OK">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font-size:14px;
border-style:solid;
border-width:2px;
background-color: rgb(47, 105, 235);
border-color:rgb(47, 105, 235);
border-radius:4px;
padding-left:15px;
padding-right:15px;</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
